src/stream/webrtc/signalling_server.rs

```diff
@@ -300,19 +300,25 @@ impl StreamManagementInterface<Stream> for SignallingServer {
 
         streams
             .iter()
-            .map(|stream| {
+            .filter_map(|stream| {
                 let (height, width, encode, interval) =
                     match &stream.video_and_stream.stream_information.configuration {
-                        crate::stream::types::CaptureConfiguration::Video(configuration) => (
-                            Some(configuration.height),
-                            Some(configuration.width),
-                            Some(format!("{:#?}", configuration.encode)),
-                            Some(
-                                (configuration.frame_interval.numerator as f32
-                                    / configuration.frame_interval.denominator as f32)
-                                    .to_string(),
-                            ),
-                        ),
+                        crate::stream::types::CaptureConfiguration::Video(configuration) => {
+                            // Filter out non-H264 local streams
+                            if configuration.encode != crate::video::types::VideoEncodeType::H264 {
+                                return None;
```

Comment:
We should also put a message saying that it was filtered.

Reply:
I can add a tracing level, otherwise, it will be shown for every webrtc's streams available request